client/post-editor/media-modal/index.jsx

```diff
@@ -342,6 +344,10 @@ export class EditorMediaModal extends Component {
 		resetAllImageEditorState();
 
 		this.props.setView( ModalViews.DETAIL );
+
+		if ( this.props.onImageEditorDoneHook ) {
```

Comment:
Check is probably overkill since we're defaulting `onImageEditorDoneHook` to `noop` anyway.

Reply:
Thanks, removed the check in dad412b8d60f1d757c7871cd1ae85c107bb8c966.